libs/slides/src/lib/arrows/slides-arrows.component.ts

```diff
@@ -17,14 +18,17 @@ export class SlidesArrowsComponent {
    */
   @HostBinding('class.shortcuts-context') context = true;
 
-  constructor(private presentation: SlidesDeckComponent) {}
+  constructor(
+    private readonly router: Router,
```

Comment:
This file can be reverted, right?

Reply:
yes... think left some changes in there that are not needed... how do i revert it in a branch. just pull the source from master and overwrite?